### ISSUE TITLE
2.8.0 Endpoint with Parameters fail in UI to be tried out...

### ISSUE THREAD

**Issue by lost-carrier** [springfox/springfox#2217]
Created a simple endpoint with some Parameter "param1", like this:
```
@RestController
@RequestMapping("/demo/")
public class DemoController {

    @GetMapping(value = "endpoint", params = "param1")
    public String endpointWithParam1(@RequestParam("param1") String param1) {
        return "blablub";
    }
}
```

The Swagger UI will display it nicely, but trying out that endpoint will fail, because curly brackets are included in the called URL:

<img width="1068" alt="screen_shot_2018-01-30_at_16_37_51" src="https://user-images.githubusercontent.com/3090579/35575271-437103cc-05dc-11e8-97f9-2127b3f0babd.png">

All versions are pretty recent:
```
	<parent>
		<groupId>org.springframework.boot</groupId>
		<artifactId>spring-boot-starter-parent</artifactId>
		<version>1.5.9.RELEASE</version>
		<relativePath/> <!-- lookup parent from repository -->
	</parent>

[...]

		<dependency>
			<groupId>io.springfox</groupId>
			<artifactId>springfox-swagger-ui</artifactId>
			<version>2.8.0</version>
		</dependency>
		<dependency>
			<groupId>io.springfox</groupId>
			<artifactId>springfox-swagger2</artifactId>
			<version>2.8.0</version>
		</dependency>
```

SwaggerConfig.java is as follows:
```
@Configuration
@EnableSwagger2
public class SwaggerConfig {

    @Bean
    public Docket api() {
        return new Docket(DocumentationType.SWAGGER_2)
                .enableUrlTemplating(true)
                .apiInfo(apiInfo())
                .select()
                .apis(RequestHandlerSelectors.basePackage("com.example.demo"))
                .paths(PathSelectors.any())
                .build();
    }

    private ApiInfo apiInfo() {
        return new ApiInfoBuilder()
                .title("Demo")
                .version("0.0.1-SNAPSHOT")
                .build();
    }

}
```

**Comment by lost-carrier** [comment]
Thanks for that hint - it works in-deed if I only have one endpoint.

Let me extend my example as follows:
```
@RestController
@RequestMapping("/demo/")
public class DemoController {

    @GetMapping(value = "endpoint", params = "param1")
    public String endpointWithParam1(@RequestParam("param1") String param1) {
        return "blablub";
    }

    @GetMapping(value = "endpoint", params = "param2")
    public String endpointWithParam2(@RequestParam("param2") String param2) {
        return "blubbla";
    }
}
```

...when I set `.enableUrlTemplating(false)`, I only get one of those endpoints in the Swagger doc:

<img width="825" alt="screen shot 2018-02-05 at 11 49 55" src="https://user-images.githubusercontent.com/3090579/35800924-bfd96638-0a6a-11e8-9e67-562fb1c4d4c3.png">

**Comment by dilipkrish** [comment]
@lost-carrier your options are
1. If you want to use overloaded methods like above, you need to set `enableUrlTemplating(true)` and use this ui library

```pom
<dependency>
	<groupId>io.springfox</groupId>
	<artifactId>springfox-swagger-ui-rf6570</artifactId>
	<version>1.0.0</version>
</dependency>
```
2. If you want the new UI then you cannot use `enableUrlTemplating(true)` as it doesnt understand url templates. And you'll end up losing one of the methods in the UI.

**Comment by w3blogfr** [comment]
Has the problem been fixed in Swagger 3.0.0? Or do we have a workaround to keep enableUrlTemplating=true and make the UI working?
